lib/components/normal-components/Switch.ts

```diff
@@ -0,0 +1,72 @@
+import { switchProps } from "@tscircuit/props"
```

Comment:
either this or switch input props

```suggestion
import { SwitchProps } from "@tscircuit/props"
```

Reply:
yes sorry. You should remove ExtendedSwitchProps- they're already defined in the tscircuit/props repo, just import them (idk the name exactly)

```suggestion
import { switchProps, SwitchProps } from "@tscircuit/props"
```